src/pages/review/detail/index.tsx

```diff
@@ -1,29 +1,61 @@
-import { IReviewData } from '@/pages/products-detail/components/review/Review.type'
+import { useState } from 'react'
+
+import { getReveiwDetail, postComment } from '@/api/reviewAPI'
 import ReviewPage from '@/pages/products-detail/components/review/ReviewPage'
-import { reviewData } from '@/pages/products-detail/mockData'
 import CommentCard from '@/pages/review/components/comment-card'
+import { comment } from '@/store/comment-slice/comment-slice'
+import { ICommentSliceState } from '@/store/comment-slice/comment-slice.type'
 import { sheet } from '@/store/sheet-slice/sheet-slice'
-import ReviewProductCard from '@components/review-product-card'
-import { useState } from 'react'
+import type { ISheetSliceState } from '@/store/sheet-slice/sheet-slice.type'
+import { RootState } from '@/store/store'
+import { ICommentData } from '@/types/getReviewDetailData.type'
 
 import BottomSheet from '@components/bottom-sheet'
 import PageHeader from '@components/page-header'
+import ReviewProductCard from '@components/review-product-card'
+import { useQuery } from '@tanstack/react-query'
 import { useDispatch, useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
 
 import * as S from './ReviewDetailPage.style'
-// eslint-disable-next-line import/order
-import type { ISheetSliceState } from '@/store/sheet-slice/sheet-slice.type'
-import { RootState } from '@/store/store'
-import dummyComment from './dummyData.json'
 
 export default function ReviewDetailPage() {
-  const [numOfComments, setNumOfComments] = useState(dummyComment.length)
-  const [inputValue, setInputValue] = useState('')
+  const { data: reviewData, refetch } = useQuery({
+    queryKey: ['review-detail'],
+    queryFn: () => getReveiwDetail(4, 1),
+  })
+  const { reviewId } = useParams()
   const dispatch = useDispatch()
   const sheetReducer = useSelector(
     (state: ISheetSliceState) => state.sheet.value,
   )
   const productDetail = useSelector((state: RootState) => state.product.detail)
+  const commentReducer = useSelector(
+    (state: ICommentSliceState) => state.comment.value,
+  )
+  const commentId = commentReducer.parentId
+
+  const [inputValue, setInputValue] = useState('')
+
+  const handleComment = (e: React.FormEvent) => {
+    e.preventDefault()
+
+    if (reviewId === undefined) return
```

Comment:
P3: 좀 더 간결하게 코드를 작성할 수 있을 거 같아요!

```suggestion
    if (!reviewId) return
```